src/main/scala/netty/ActorChannelConnector.scala

```diff
@@ -10,12 +10,15 @@ import org.apache.pekko.actor.typed.ActorRef
 import lila.ws.Controller.Endpoint
 import lila.ws.netty.ProtocolHandler.key
 
-final private class ActorChannelConnector(clients: ActorRef[Clients.Control])(using Executor):
+final private class ActorChannelConnector(clients: ActorRef[Clients.Control], loop: WorkerLoop)(using
+    Executor
+):
 
   def apply(endpoint: Endpoint, channel: Channel): Unit =
     val clientPromise = Promise[Client]()
     channel.attr(key.client).set(clientPromise.future)
-    val channelEmit = emitToChannel(channel)
+    val channelEmit: ClientEmit =
+      emitToChannel(channel, withFlush = endpoint.name == "round/play")
```

Comment:
abstraction leak

Reply:
haha you said abstraction

Reply:
you thinking we set attr("shapedFlush") or something in the channel context on creation and then check that?